proto/common.proto

```diff
@@ -431,25 +431,25 @@ message RaidBuffs {
 
 	// 10% AP
 	bool trueshot_aura = 11;
-	bool unleashed_rage = 12;
-	bool abominations_might = 13;
+	TristateEffect unleashed_rage = 12;
+	TristateEffect abominations_might = 13;
```

Comment:
we use tristate effects for buffs that can be improved with a talent and not for points leveled. We always assume when external those talents are at max points. I think we want this to be back to bool and assume max bonus

Reply:
so that would mean you could never sim if the dk in your raid only has one point in it?